athletics.lic

```diff
@@ -336,7 +336,8 @@ class Athletics
   def climb_branch
     if UserVars.athletics < 540
       override_location_and_practice('undergondola_branch')
-    elsif UserVars.athletics < 850
+    else
+      DRC.message("Warning: ;athletics undergondola with more than 850 athletics is not best use and you may consider ;athletics wyverns instead.") if UserVars.athletics > 850
```

Comment:
Love the anti-death change ☠️ 🐉 

💡 For Genie users where semicolons are not shown, suggest using `#{$lich_char}`. Not a blocker for merge, though.
```suggestion
      DRC.message("Warning: '#{$lich_char}athletics undergondola' with more than 850 athletics is not best use and you may consider '#{$lich_char}athletics wyverns' instead.") if UserVars.athletics > 850
```

Reply:
Good catch, but since this involved character deaths I'm going to merge it immediately. I would really like to see the cleanup PR, though.